### PR TITLE
Add GoCenter as GOPROXY to speed up go builds

### PR DESCRIPTION
**Pull request by rimusz**
Signed-off-by: rimas <rmocius@gmail.com>

Using https://gocenter.io as GOPROXY will speed modules download and will ensure that modules referenced in `go.mod` are always the same, doesn't matter if some modules get deleted from GitHub and etc, they got cached for live at gocenter.io :-)

### CODE REVIEW

**Comment by rimusz** [comment]
cc @migmartri @prydonius

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,44 @@
+module github.com/rimusz/monocular
```
Comment (prydonius) [inline go.mod]: guess this should be github.com/helm/monocular?

Comment (rimusz) [reply go.mod]: yes, autogeneration, will fix it up

Comment (rimusz) [reply go.mod]: done

**Comment by prydonius** [comment]
This is really great, thanks @rimusz. However, would you also be able to update the developer documentation to replace the dep references with the relevant go module commands? https://github.com/helm/monocular/blob/master/docs/development.md#developers-guide

**Comment by rimusz** [comment]
sure

**Pull request merged**